guides/compatibility-mode.md

```diff
@@ -0,0 +1,8 @@
+## What is compatibility mode?
+Compatibility mode for `@angular/material` allows the components to exist side-by-side with AngularJS Material components without any CSS collisions.
+
+## What does compatibility mode do?
+When enabled, compatibility mode enforces that all template APIs use the prefix `mat-` instead of `md-`. This will prevent any CSS from AngularJS Material from affecting the components in `@angular/material`.
```

Comment:
Maybe add an example to this section that shows, say, a button with a tooltip before and after compatibility mode is enabled?